<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveFile</class>
 <widget class="QMainWindow" name="ReceiveFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReceiveFile</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="StatusText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>721</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>496</y>
      <width>721</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="listenBtn">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>监听</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnStop">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>721</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>文件接收进度条</string>
    </property>
    <widget class="QProgressBar" name="recvProg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>691</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>监听端口号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLineEdit">
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>721</width>
      <height>76</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>选择接收文件的存储路径</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="directoryComboBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="selectDirBtn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
